evm/db/chain.py

```diff
@@ -250,15 +255,28 @@ def get_block_header_by_hash(self, block_hash: Hash32) -> BlockHeader:
                 encode_hex(block_hash)))
         return _decode_block_header(header_rlp)
 
-    def header_exists(self, block_hash: Hash32) -> bool:
-        """Returns True if the header with the given block hash is in our DB."""
-        return self.db.exists(block_hash)
+    def get_block_uncles(self, uncles_hash: Hash32) -> List[BlockHeader]:
+        """
+        Returns an iterable of uncle headers specified by the given uncles_hash
+        """
+        validate_word(uncles_hash, title="Uncles Hash")
+        return rlp.decode(self.db.get(uncles_hash), sedes=rlp.sedes.CountableList(BlockHeader))
+
+    def get_score(self, block_hash: Hash32) -> int:
+        """
+        Returns the score for the header with the given hash.
+
+        Raises HeaderNotFound if no header with the given has is found in the database.
+        """
+        return rlp.decode(
+            self.db.get(SchemaV1.make_block_hash_to_score_lookup_key(block_hash)),
+            sedes=rlp.sedes.big_endian_int)
 
-    # TODO: This method sould take a chain of headers as that's the most common use case
+    # TODO: This method should take a chain of headers as that's the most common use case
     # and it'd be much faster than inserting each header individually.
     def persist_header(self, header: BlockHeader) -> Tuple[BlockHeader, ...]:
         """
-        :returns: iterable of headers newly on the canonical chain
+        Returns iterable of headers newly on the canonical chain
```

Comment:
Why drop the Sphinx `:return:` decorator?

Reply:
Consistency.  A majority of the other methods weren't using this format so I decided to make them all the same.  I figure we can go through and convert these to *all* use sphinx formatting in a subsequent PR if that is how we're documenting things.